config/features/config.go

```diff
@@ -214,9 +214,10 @@ func ConfigureBeaconChain(ctx *cli.Context) error {
 		logEnabled(prepareAllPayloads)
 		cfg.PrepareAllPayloads = true
 	}
-	if ctx.IsSet(buildBlockParallel.Name) {
-		logEnabled(buildBlockParallel)
-		cfg.BuildBlockParallel = true
+	cfg.BuildBlockParallel = true
+	if ctx.IsSet(disableBuildBlockParallel.Name) {
+		logEnabled(disableBuildBlockParallel)
+		cfg.BuildBlockParallel = false
```

Comment:
Could we not rename this to `DisableBuildBlockParallel`, that way it is false by default and then just set it to true if the flag `disableBuildBlockParallel` is set? Would also have to invert the condition where it is used in the code as well. But basically:
```suggestion
        if ctx.IsSet(disableBuildBlockParallel.Name) {
                logEnabled(disableBuildBlockParallel)
                cfg.DisableBuildBlockParallel = true
```
It seems clearer than `disableBuildBlockParallel` setting `BuildBlockParallel` to false

Reply:
This has been the standard pattern: https://github.com/prysmaticlabs/prysm/pull/9225

Reply:
Understood, thank you for clarifying!